pkg/tuf/keys.go

```diff
@@ -24,9 +24,9 @@ import (
 func getPassphraseRetriever() notary.PassRetriever {
 	baseRetriever := passphrase.PromptRetriever()
 	env := map[string]string{
-		"root":     os.Getenv("SIGNY_ROOT_PASSPHRASE"),
-		"targets":  os.Getenv("SIGNY_TARGETS_PASSPHRASE"),
-		"releases": os.Getenv("SIGNY_RELEASES_PASSPHRASE"),
+		"root":             os.Getenv("SIGNY_ROOT_PASSPHRASE"),
```

Comment:
As we think more about how consumers will end up using this as a library rather than a CLI, we should think about making the environment variables for setting passphrases to be configurable.

Reply:
Opened #82 to track this.